<launch>
  <arg name="START_X" default="0.0"/>
  <arg name="START_Y" default="0.0"/>
  <arg name="WAYPOINT_DISTANCE" default="1.5"/>
  <arg name="MARGIN_WALL" default="0.3"/>

  <node pkg="chibi18_control" name="global_path_planner" type="global_path_planner">
    <param name="START_X" value="$(arg START_X)"/>
    <param name="START_Y" value="$(arg START_Y)"/>
    <param name="WAYPOINT_DISTANCE" value="$(arg WAYPOINT_DISTANCE)"/>
    <param name="MARGIN_WALL" value="$(arg MARGIN_WALL)"/>
  </node>

</launch>
